xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecCCText.cpp

```diff
@@ -123,6 +123,14 @@ OverlayMessage CDVDOverlayCodecCCText::Decode(DemuxPacket* pPacket)
   return m_pOverlay ? OverlayMessage::OC_DONE : OverlayMessage::OC_OVERLAY;
 }
 
+void CDVDOverlayCodecCCText::PostProcess(std::string& text)
+{
+  // The data that come from InputStream could contains \r chars
+  // we have to remove them all because it causes to display empty box "tofu"
+  //! @todo This must be removed after the rework of the CC decoders
+  StringUtils::Replace(text, "\r", "");
```

Comment:
I'd say any implementation of `PostProcess` in derived classes should call base just for the sake of it (although it currently does nothing)

Reply:
done